recipes/apache2.rb

```diff
@@ -0,0 +1,53 @@
+#
+# Cookbook Name:: cachethq
+# Recipe:: apache2
+#
+# All rights reserved - Do Not Redistribute
+#
+
+cachethq = node.default['cachethq']
+apache2 = cachethq['apache2']
+
+# somehow apache2 is also installed
+package %w(libapache2-mod-auth-cas apache2)  do
+  action :install
+end
+
+execute "a2enmod auth_cas" do
+  command "sudo a2enmod auth_cas"
+end
+
+default_conf = "#{apache2['confdir']}" + "000-default.conf"
```

Comment:
is it better to make this default attributes, not hardcoded

Reply:
done please check Mas.